sdk/src/main/java/ly/count/android/sdk/ConnectionProcessor.java

```diff
@@ -191,14 +191,20 @@ synchronized public URLConnection urlConnectionForServerRequest(String requestDa
         }
 
         //calculating header field size
-        int headerIndex = 0;
-        while (true) {
-            String key = conn.getHeaderFieldKey(headerIndex);
-            if (key == null) {
-                break;
+        try{ //just after init, because of background operations, this might fail
```

Comment:
comments should be in new line... change `try{` to `try {`

Reply:
done